### PR TITLE
Sync in O3 agama repo for s390x

### PR DESCRIPTION
**Pull request by lemon-suse**
Related ticket: https://progress.opensuse.org/issues/166253

### CODE REVIEW

File: xml/obs/systemsmanagement:Agama:Devel.xml [hunk 1]
```diff
@@ -3,8 +3,9 @@
     dist_path="obspublish-other::openqa/systemsmanagement:Agama:Devel/"
     distri="opensuse"
     version="agama-9.0"
-    archs="x86_64 aarch64 ppc64le s390x">
+    archs="x86_64 aarch64 ppc64le">
     <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
         <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
     </flavor>
+    <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" archs='s390x' iso="extract_as_repo"/>
```
Comment (andrii-suse) [inline xml/obs/systemsmanagement:Agama:Devel.xml:10]: hm, I think when adding `<flavor>` with the same name it just overwrites the previous one. This is why x86_64 iso  is not in print_rsync_iso output anymore

File: t/obs/systemsmanagement:Agama:Devel/base/files_iso
Comment (andrii-suse) [inline t/obs/systemsmanagement:Agama:Devel/base/files_iso:1]: I believe this file is accidental - just delete it?

File: t/obs/systemsmanagement:Agama:Devel/files_iso.lst [hunk 2]
```diff
@@ -1,5 +1 @@
-agama-installer.aarch64-9.0.0-openSUSE-Build8.7.iso
-agama-installer.ppc64le-9.0.0-openSUSE-Build8.7.iso
-agama-installer.s390x-9.0.0-openSUSE-Build8.7.iso
-agama-installer.x86_64-9.0.0-openSUSE-Build8.7.iso
-
+agama-installer.s390x-9.0.0-openSUSE-Build17.9.iso
```
Comment (andrii-suse) [inline t/obs/systemsmanagement:Agama:Devel/files_iso.lst:1]: all the rest files in `t`, which are not in `base` nor `s390x` folder should be removed if you add batches.

**Review by andrii-suse — APPROVED** [review]

File: xml/obs/systemsmanagement:Agama:Devel.xml [hunk 3]
```diff
@@ -2,9 +2,13 @@
...
     distri="opensuse"
-    version="agama-9.0"
-    archs="x86_64 aarch64 ppc64le s390x">
-    <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
-        <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
-    </flavor>
+    version="agama-9.0">
+    <batch name="base" archs="86_64 aarch64 ppc64le">
+        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
+            <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
```
Comment (andrii-suse) [inline xml/obs/systemsmanagement:Agama:Devel.xml:8]: Do you really need hdd? I expected it should work without it as well

File: t/obs/systemsmanagement:Agama:Devel/base/files_iso
Comment (lemon-suse) [reply t/obs/systemsmanagement:Agama:Devel/base/files_iso:1]: Thanks, removed it.

File: t/obs/systemsmanagement:Agama:Devel/files_iso.lst (hunk 2 above)
Comment (lemon-suse) [reply t/obs/systemsmanagement:Agama:Devel/files_iso.lst]: Yes, removed.

File: xml/obs/systemsmanagement:Agama:Devel.xml (hunk 3 above)
Comment (lemon-suse) [reply xml/obs/systemsmanagement:Agama:Devel.xml]: I updated it following your suggestion, I think this will be more simple and clear. :)

**Pull request merged**